### ISSUE TITLE
strikethrough

### ISSUE THREAD

**Issue by leanhforevo** [song940/node-escpos#165]
hello!
please show me the way to write text StrikeThrough

**Comment by Psychopoulet** [comment]
printer.style("u")
b => bold
u => underline
i => italic
combo (bu, bi, iu, etc...)

we should put it on the doc, I guess

**Comment by aroms** [comment]
Yes, please, add it to the documentation, because the comment you have put I do not understand it, and I would go great this option for a project with which I am right now ...

Thank you.

**Comment by Psychopoulet** [comment]
that's it (sorry for confusion about arrow functions)
there is no StrikeThrough, but you can use some other styles

**Comment by brkrtp** [comment]
+1
